docs/source/docker/advanced/datasource_metadata.rst

```diff
@@ -0,0 +1,177 @@
+
+
+
+
+Metadata Schema
+---------------
+The schema defines the format of the metadata and helps the Lightly Platform to correctly identify 
+and display different types of metadata.
+
+You can provide this information to Lightly by adding a `schema.json` to the 
+`.lightly/metadata` directory. The `schema.json` file must contain a list of
+configuration entries. Each of the entries is a dictionary with the following keys:
+
+ - `name`: Identifier of the metadata in the UI.
+ - `path`: Concatenation of the keys to access the metadata in a dictionary.
+ - `defaultValue`: The fallback value if there is no metadata available.
+ - `valueDataType`: One of
+
+   - `NUMERIC_INT`
+   - `NUMERIC_FLOAT`
+   - `CATEGORICAL_INT`
+   - `CATEGORICAL_STRING`
+   - `CATEGORICAL_BOOLEAN`
+
+
+For example, let's say we have additional information about the weather for each
+of the images we have collected. A possible schema could look like this:
+
+.. code-block:: javascript
+    :caption: .lightly/metadata/schema.json
+
+    [
+        {
+            "name": "Is special frame",
+            "path": "special_frame_flag",
+            "defaultValue": false,
+            "valueDataType": "CATEGORICAL_BOOLEAN"
+        },
+        {
+            "name": "Weather description",
+            "path": "weather.description",
+            "defaultValue": "nothing",
+            "valueDataType": "CATEGORICAL_STRING"
+        },
+        {
+            "name": "Temperature",
+            "path": "weather.temperature",
+            "defaultValue": 0.0,
+            "valueDataType": "NUMERIC_FLOAT"
+        },
+        {
+            "name": "Air pressure",
+            "path": "weather.air_pressure",
+            "defaultValue": 0,
+            "valueDataType": "NUMERIC_INT"
+        },
+        {
+            "name": "Vehicle ID",
+            "path": "vehicle_id",
+            "defaultValue": 0,
+            "valueDataType": "CATEGORICAL_INT"
+        }
+    ]
+
+
+
+
+Metadata Files
+--------------
+Lightly requires a single metadata file per image or video. If a metadata file is provided
+for a full video, Lightly assumes that the metadata is valid for all frames in that video.
+
+To provide metadata for an image or a video, place a metadata file with the same name
+as the image or video in the `.lightly/metadata` directory but change the file type to
+`.json`. The file should contain the metadata in the format defined under :ref:`ref-metadata-format`.
+
+
+.. code-block:: bash
+
+    # filename of the metadata for file FILENAME.EXT
+    .lightly/metadata/${FILENAME}.json
+
+    # example: my_image.png
+    .lightly/metadata/my_image.json
+
+    # example: my_video.mp4
+    .lightly/metadata/my_video.json
+
+
+When working with videos it's also possible to provide metadata on a per-frame basis.
+Then, Lightly requires a metadata file per frame. Lightly uses a naming convention to
+identify frames: The filename of a frame consists of the video filename, the video format,
```

Comment:
since the video format is at the end of the naming convention, mention it here at the end